src/Entity/User.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+// Totalement inutile car remplacé par User2.php
+
+namespace App\Entity;
+
+use App\Repository\UserRepository;
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+
+#[ORM\Entity(repositoryClass: UserRepository::class)]
+#[UniqueEntity('email', message: 'Cet e-mail est déjà utilisé.')]
+class User
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column]
+    private ?int $id = null;
```

Comment:
Un ID ne peut jamais être null. Donc ton auttribut aussi
```suggestion
    private int $id = 1;
```